config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3027,6 +3027,11 @@
 # Default: false 
 # samigo.wysiwyg.delivery.disable=true
 
+# SAM-1947 Email notifications on Samigo Submission
+# The from address for individual submission notifications
+# DEFAULT: no-reply@serverName
+# samigo.fromAddress=<SAMIGO_SMTP_FROM>
+
```

Comment:
I added in the configuration option which allows users to set the address individual submission email notifications are sent from.